Updated Asterisk, Chibi, Micro

Colin-Tel opened the pull request:
<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->

## About the PR
<!-- What did you change? -->
Updated maps to be more in-line with current map rules/content.
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Regular map maintenance
## Technical details
<!-- Summary of code changes for easier review. -->
YAML
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. -->
N/A
## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->
N/A
**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelogs must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- tweak: Updated Asterisk and Chibi (and Micro) to have department techfabs.

Colin-Tel committed:
Updated Asterisk, Chibi, Micro

Updated maps to be more in-line with current map rules/content.

FieldCommand commented:
Hello there.
Could you put Micro in a separate PR. So Asterisk and Chiby can be merged right away as they are good to go. I need more time with Micro, as that has much more substantial work to it.

Colin-Tel commented:
@FieldCommand 
Does it necessarily make a difference even if Micro is out of rotation?

FieldCommand commented:
Keeping Micro up to date so it doesn't fall behind for when SM eventually gets implemented is all good, and I do approve.

But to me, it sort of doesn't fit with the PR of just keeping chibi and asterisk map compliant, and Micro also has some huge changes to it, and I have no idea what has changed besides me just comparing files locally, which is time-consuming, and I would like to practice going over map changes so I can get better at it. Even still, if it's not in rotation. And two other maps could get merged right away while I took my time on Micro.

Regardless, I will deal either way @Colin-Tel :)

IamVelcroboy commented:
@Colin-Tel can you list the changes you made in the PR description for Micro since the diff log is too big to load. That way we at least know where to look for changes. Please/thanks 😘

Colin-Tel commented:
> @Colin-Tel can you list the changes you made in the PR description for Micro since the diff log is too big to load. That way we at least know where to look for changes. Please/thanks 😘

1. added turrets to AI sat
2. added holopads
3. replaced lathes with techfabs
4. added ammo techfab to armory, sec techfab in their locker room
5. added circuit imprinter to eng
6. removed herobrine
